packages/core/src/dialects/abstract/query-interface.d.ts

```diff
@@ -467,6 +477,14 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
     options?: AddConstraintOptions & QueryRawOptions
   ): Promise<void>;
 
+  /**
+  * Show constraints on a table
+  */
+  showConstraint(
+    tableName: TableName,
+    options?: { tableName: TableName }
```

Comment:
This signature does not match the implementation of showConstraint, which takes `tableName: TableName`, `constraintName?: string` & `options?: QueryOptions`

Reply:
Done